### ISSUE TITLE
[Accessibility] date-picker doesn't have tooltip on previous, next button

### ISSUE THREAD

**Issue by ipadthree** [angular/components#16133]
#### Reproduction
Steps to reproduce:
1. Go to date-picker example page: https://material.angular.io/components/datepicker/overview
2. Open a date-picker
3. Hover over the arrow buttons on top right of modal
 
 
#### Expected Behavior

What behavior were you expecting to see?
Tooltip with 'Next' appears on `>` arrow button
Tooltip with 'Previous' appears on `<` arrow button

#### Actual Behavior

What behavior did you actually see?
No tooltip is shown.

#### Environment

- Angular:
- CDK/Material:
- Browser(s):
- Operating System (e.g. Windows, macOS, Ubuntu):

**Comment by jelbourn** [comment]
These buttons both have an `aria-label`. AFAIK the presence of a tooltip is not tied to a11y.
